### PR TITLE
Add slack #summit channel link

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

N/A

### Description of the Change

Added link to slack for the summit.

### Alternate Designs

None

### Benefits

A visible place for questions

### Possible Drawbacks

None

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias Persson <tobias.persson@axis.com>

### CODE REVIEW

File: summit.html [hunk 1]
```diff
@@ -26,9 +26,9 @@
       <div>
         <h1 class="section-heading">The practicals</h1>
         <p class="section-paragraph">
-	  <ul>
-	    <li>Where: <a target="_blank" href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_NGM1NDM4OGYtYzNmZi00OGY2LTlmODEtZmMyMmY0MDRkMWYy%40thread.v2/0?context=%7b%22Tid%22%3a%2292e84ceb-fbfd-47ab-be52-080c6b87953f%22%2c%22Oid%22%3a%2249725723-aa8c-4dcf-b9d0-b974e8a25702%22%7d">Microsoft Teams Meeting</a></li>
-	    <li>When : 22nd–23rd of September, 9:00–16:00 CEST</li>
+    <ul>
+      <li>Where: <a target="_blank" href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_NGM1NDM4OGYtYzNmZi00OGY2LTlmODEtZmMyMmY0MDRkMWYy%40thread.v2/0?context=%7b%22Tid%22%3a%2292e84ceb-fbfd-47ab-be52-080c6b87953f%22%2c%22Oid%22%3a%2249725723-aa8c-4dcf-b9d0-b974e8a25702%22%7d">Microsoft Teams Meeting</a></li>
+      <li>When : 22nd–23rd of September, 9:00–16:00 CEST</li>
```
Comment (magnusbaeck) [inline summit.html:31]: Thou hath messed up the indentation.

File: summit.html [hunk 2]
```diff
@@ -38,8 +38,11 @@ <h1 class="section-heading">The practicals</h1>
               Who: To get in touch with the organizers send an email to
               <a href="mailto:eiffel-tc@googlegroups.com">eiffel-tc@googlegroups.com</a>.
             </li>
-	  </ul>
-	</p>
+            <li>
+              Questions: To ask questions during, after or before the summit please join us on <a href="https://eiffel-workspace.slack.com/archives/C02EKUS4M24">slack</a>.
```
Comment (magnusbaeck) [inline summit.html:42]: Well, we don't have to limit ourselves to questions.

**Review by magnusbaeck — APPROVED** [review]

File: summit.html (hunk 2 above)
Comment (m-linner-ericsson) [reply summit.html:42]: If we want to capture not just questions, would the following work?

```suggestion
              Communication: For questions, comments, or suggestions regarding the Summit, please join us on <a href="https://eiffel-workspace.slack.com/archives/C02EKUS4M24">slack</a>.
```

Comment (magnusbaeck) [reply summit.html:42]: Sure, that'd work.

File: summit.html (hunk 1 above)
Comment (t-persson) [reply summit.html:31]: The indentation was mixed tabs and spaces.
I merely did "s/\t/  /g" :)

**Review by magnusbaeck — APPROVED** [review]

**Review by m-linner-ericsson — APPROVED** [review]

**Pull request merged**